### PR TITLE
Add Swift Package Manager support

### PR DESCRIPTION
**Pull request by raxityo**
Thanks for sharing this library @HeshamMegid.

I am using SPM for my projects and figured it will be valuable for others who are using SPM if you add Package.swift to this project.

### CODE REVIEW

**Comment by tc-sgupta** [comment]
@HeshamMegid Could you please review this PR. 
https://github.com/HeshamMegid/HMSegmentedControl/issues/354

**Review by tc-sgupta — DISMISSED** [review]

**Comment by tc-sgupta** [comment]
@raxityo Could you please add a support for swift-tools-version:5.0, 5.1, 5.2.

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.3
```
Comment (tc-sgupta) [inline Package.swift:1]: If we change 5.3 to 5.0 then xcode 11 can use SPM to add this framework

Comment (raxityo) [reply Package.swift]: Good call. I only tested in Xcode 12. Downgraded in 47d7744. 👍

**Review by tc-sgupta — APPROVED** [review]

**Review by HeshamMegid — APPROVED** [review]

**Comment by HeshamMegid** [comment]
Thanks @raxityo!

**Comment by tc-sgupta** [comment]
@HeshamMegid Could you please create a new release

**Comment by o15a3d4l11s2** [comment]
Trying to use SPM for installation yields an error
_https://github.com/HeshamMegid/HMSegmentedControl has no Package.swift manifest for version 1.5.6_
because there is no release version that contains the Package.swift file. @HeshamMegid , could you please prepare a new release?

**Comment by sgraesser17** [comment]
@HeshamMegid, I too would like to see a new release (v1.5.7?) that includes this fix.

File: .gitignore [hunk 2]
```diff
@@ -14,4 +14,5 @@ profile
 *.moved-aside
 DerivedData
 .idea/
-*.hmap
```
Comment (mervyd) [inline .gitignore:17]: Package.swift

**Comment by mervyd** [comment]
#352

**Pull request merged**